docs/source/array-creation.rst

```diff
@@ -13,9 +13,10 @@ their arrangement can be as simple or as complex as the situation dictates.
 Simple case - Format Supports NumPy Slicing
 -------------------------------------------
 
-Many formats have Python projects that expose storage using NumPy slicing syntax.
-For example the HDF5 file format has the ``h5py`` project, which provides a
-``Dataset`` object into which we can slice in NumPy fashion
+Many storage formats have Python projects that expose storage using NumPy
+slicing syntax.  These include HDF5, NetCDF, BColz, Zarr, GRIB, etc..  For
```

Comment:
I believe that double period on etc. is common when ending a sentence with etc..